docs/admin_api/media_admin_api.md

```diff
@@ -100,3 +100,27 @@ Response:
   "num_quarantined": 10  # The number of media items successfully quarantined
 }
 ```
+
+# Draft: Delete local media
+This API deletes the *local* media from the disc of your own server.
+
```

Comment:
I think it's also important to mention that this API will not affect media that has been uploaded to external media repositories (aka https://github.com/turt2live/matrix-media-repo/).